### ISSUE TITLE
windows users no longer return tty

### ISSUE THREAD

**Issue by si458** [sebhildebrandt/systeminformation#643]
**Describe the bug**
windows users no longer return tty after switching to powershell

**To Reproduce**
1. `si.users()`

**Current Output**
If applicable, add output to help explain your problem.
```
[
  {
    user: 'Simon',
    tty: '',
    date: '2022-01-18',
    time: '20:19',
    ip: '',
    command: ''
  }
]
```

**Expected behavior**
```
[
  {
    user: 'Simon',
    tty: 'console',
    date: '2022-01-18',
    time: '20:19',
    ip: '',
    command: ''
  }
]
```
**Environment (please complete the following information):**
 - systeminformation package version: any verson above 5.9.7
 - OS: [e.g. macOS] Windows 10 Pro
 - Hardware [e.g. MacBook Pro 13] Parallels Desktop on Macbook Pro OR phsycial Dell PC

**Additional context**

the docs page still shows windows will return the tty value - https://systeminformation.io/os.html

also it might be better to return to using the `query user` in powershell or cmd as its 10x quicker personally

**Comment by si458** [comment]
you could also maybe set `tty` to the `sessionid`
e.g: this allows me to set the shadow id to shadow an rdp session remotely
e.g: `mstsc /shadow:{TTY} /v:192.168.1.100 /control /noConsentPrompt`

you can retrieve the sessionid by querying the explorer.exe process
`Get-WmiObject Win32_Process -Filter "name='explorer.exe'"`
```
_GENUS                     : 2
__CLASS                    : Win32_Process
__SUPERCLASS               : CIM_Process
__DYNASTY                  : CIM_ManagedSystemElement
__RELPATH                  : Win32_Process.Handle="6588"
__PROPERTY_COUNT           : 45
__DERIVATION               : {CIM_Process, CIM_LogicalElement, CIM_ManagedSystemElement}
__SERVER                   : SIWIN10MAC
__NAMESPACE                : root\cimv2
__PATH                     : \\SIWIN10MAC\root\cimv2:Win32_Process.Handle="6588"
Caption                    : explorer.exe
CommandLine                : C:\Windows\Explorer.EXE
CreationClassName          : Win32_Process
CreationDate               : 20220118201907.713816+000
CSCreationClassName        : Win32_ComputerSystem
CSName                     : SIWIN10MAC
Description                : explorer.exe
ExecutablePath             : C:\Windows\Explorer.EXE
ExecutionState             :
Handle                     : 6588
HandleCount                : 2968
InstallDate                :
KernelModeTime             : 1433281250
MaximumWorkingSetSize      : 1380
MinimumWorkingSetSize      : 200
Name                       : explorer.exe
OSCreationClassName        : Win32_OperatingSystem
OSName                     : Microsoft Windows 10 Pro|C:\Windows|\Device\Harddisk0\Partition4
OtherOperationCount        : 450771
OtherTransferCount         : 11791015
PageFaults                 : 491249
PageFileUsage              : 111908
ParentProcessId            : 6560
PeakPageFileUsage          : 116588
PeakVirtualSize            : 2204345368576
PeakWorkingSetSize         : 183012
Priority                   : 8
PrivatePageCount           : 114593792
ProcessId                  : 6588
QuotaNonPagedPoolUsage     : 772
QuotaPagedPoolUsage        : 1748
QuotaPeakNonPagedPoolUsage : 772
QuotaPeakPagedPoolUsage    : 2248
ReadOperationCount         : 187201
ReadTransferCount          : 116357833
SessionId                  : 1
Status                     :
TerminationDate            :
ThreadCount                : 74
UserModeTime               : 946875000
VirtualSize                : 2204061061120
WindowsVersion             : 10.0.19043
WorkingSetSize             : 161349632
WriteOperationCount        : 12983
WriteTransferCount         : 7015637
PSComputerName             : SIWIN10MAC
ProcessName                : explorer.exe
Handles                    : 2968
VM                         : 2204061061120
WS                         : 161349632
Path                       : C:\Windows\Explorer.EXE
```

**Comment by si458** [comment]
ive made a temp patch for people to try and see what they think its its helpful or not
i do miss the old `console` and `rdp-tcp#12` tho as it helped tell if it was a physical user or a remote user

**Comment by sebhildebrandt** [comment]
@si458 thank you Simon! I will have a look on it this weekend.

**Comment by sebhildebrandt** [comment]
@si458 Thank you once again!! I looked at it and merged it, I also made some improvements trying to get the `console` back. I pushed my code here, maybe you can give it a try. If this works, I will publish a new release with this changes.

**Comment by sebhildebrandt** [comment]
@si458 I just released a new version 5.10.6 that should fix the problem.

**Comment by si458** [comment]
thanks @sebhildebrandt, it does indeed works a treat now!
im just trying to decide if maybe we could seperate the `tty` into maybe `tty` and `rdp`?
because it currently returns `console` as expected for physical users 
BUT then RDP users return it as `rdp-tcp#18` but the session ID to watch an RDP session is different?
```
PS C:\Users\Administrator.FTH\Downloads> node .\test.js
[
  {
    user: 'Administrator',
    tty: 'rdp-tcp#18',
    date: '2022-01-21',
    time: '11:54',
    ip: '',
    command: ''
  }
]
PS C:\Users\Administrator.FTH\Downloads> query user
 USERNAME              SESSIONNAME        ID  STATE   IDLE TIME  LOGON TIME
>administrator         rdp-tcp#18          2  Active          .  15/01/2022 08:55
PS C:\Users\Administrator.FTH\Downloads>
```
